### PR TITLE
Added ticket purchase page

### PR DESCRIPTION
**Pull request by HamidNE**

### CODE REVIEW

**Comment by HamidNE** [comment]
Hi @pi0
Please check the codes and merge if possible.

File: pages/ticket.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <div>
+    <image-header />
+    <div class="container py-7">
+      <div class="contents">
+        <h4>مختصری درباره گردهمایی Vue.js</h4>
+        <p>لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ و با استفاده از طراحان گرافیک است. چاپگرها و متون بلکه روزنامه و مجله در ستون و سطرآنچنان که لازم است و برای شرایط فعلی تکنولوژی مورد نیاز و کاربردهای متنوع با هدف بهبود ابزارهای کاربردی می باشد. کتابهای زیادی در شصت و سه درصد گذشته، حال و آینده شناخت فراوان جامعه و متخصصان را می طلبد تا با نرم افزارها شناخت بیشتری را برای طراحان رایانه ای علی الخصوص طراحان خلاقی و فرهنگ پیشرو در زبان فارسی ایجاد کرد. در این صورت می توان امید داشت که تمام و دشواری موجود در ارائه راهکارها و شرایط سخت تایپ به پایان رسد وزمان مورد نیاز شامل حروفچینی دستاوردهای اصلی و جوابگوی سوالات پیوسته اهل دنیای موجود طراحی اساسا مورد استفاده قرار گیرد.</p>
```
Comment (pi0) [inline pages/ticket.vue]: Is it to be published? :D

File: utils/schema/event.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      postalCode: "19019",
+      addressRegion: "TH",
+      addressCountry: "IR"
+    }
+  },
+  image: ["https://vuejsconf.ir/img/poster2.jpg"],
+  description: "The First Vue.js Tehran Meetup",
+  offers: {
+    "@type": "Offer",
+    url: "https://www.example.com/event_offer/12345_201803180430",
```
Comment (pi0) [inline utils/schema/event.js]: Is it a valid link?

File: utils/schema/website.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export default {
+  "@context": "https://schema.org",
+  "@type": "WebSite",
+  "url": "https://vuejsconf.ir",
+  "potentialAction": {
+    "@type": "SearchAction",
+    "target": "https://vuejsconf.ir/search/?q={search_term_string}",
```
Comment (pi0) [inline utils/schema/website.js]: Do we have search page?

Comment (HamidNE) [reply utils/schema/website.js]: We do not have a search page but this is required for schema.

File: utils/schema/event.js (hunk 2 above)
Comment (HamidNE) [reply utils/schema/event.js]: Sorry, this has been fixed in the new version where the venue was designated.

File: pages/ticket.vue (hunk 1 above)
Comment (HamidNE) [reply pages/ticket.vue]: I got the correct text from @negarjf and corrected it.

File: utils/schema/website.js (hunk 3 above)
Comment (pi0) [reply utils/schema/website.js]: But an invalid 404 route is not best idea. What if we point it to google.com/?q=

Comment (HamidNE) [reply utils/schema/website.js]: I modified the correct search link.

**Pull request merged**